config/base.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  root: true,
+
+  parser: 'vue-eslint-parser',
+
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  env: {
+    browser: true,
```

Comment:
Personally, I dislike `browser` environment because this environment includes many global variables which has general names: `name`, `length`, `status`, `event`, `open`, `parent`, etc.... It will possibly cause the problems about `no-undef` / `no-unused-vars` / `no-shadow` rules.

So I prefer that it defines fewer global variables than `browser` environment manually (then I use `window.name` instead of `name`) ([browser.js](https://github.com/mysticatea/eslint-config/blob/8d348f853045eeab9b01dd8662a1fe6efa5170a6/browser.js) is my config for frontend. This defines classes and whitelisted variables of `browser` environment).

However, I know I'm in the minority.